packages/addons-v5/commands/addons/create.js

```diff
@@ -40,8 +41,19 @@ function * run (context, heroku) {
 
   let { name, as } = flags
   let config = parseConfig(args.slice(1))
+  let addon
 
-  let addon = yield createAddon(heroku, app, args[0], context.flags.confirm, context.flags.wait, { config, name, as })
+  try {
+    addon = yield createAddon(heroku, app, args[0], context.flags.confirm, context.flags.wait, { config, name, as })
+    if (context.flags.wait) {
+      notify(`heroku addons:create ${addon.name}`, 'Add-on successfully provisioned')
+    }
+  } catch (error) {
+    if (context.flags.wait) {
+      notify(`heroku addons:create ${args[0]}`, 'Add-on failed to provision', false)
```

Comment:
Technically this is a bit pessimistic: it looks like `addons_wait` does not retry, so an error here could just mean, e.g., exceeding a rate limit or a temporary failure to reach the heroku API. I think properly reasoning about possible failures through this whole chain could get pretty complicated pretty fast, but a message claiming we failed to provision when maybe we didn't could be confusing =/

Reply:
@uhoh-itsmaciek Perhaps we could use a more generic message text that says something along the lines of `Error occurred while trying to provision add-on`.  Unfortunately, if we're too generic, the customer won't know what kind of action to take.  Would using the `error.message` be sufficient?